src/sentry/sentry_apps/api/endpoints/installation_external_issues.py

```diff
@@ -45,22 +44,14 @@ def post(self, request: Request, installation) -> Response:
 
         serializer = PlatformExternalIssueSerializer(data=request.data)
         if serializer.is_valid():
-            try:
-                external_issue = ExternalIssueCreator(
-                    install=installation,
-                    group=group,
-                    web_url=data["webUrl"],
-                    project=data["project"],
-                    identifier=data["identifier"],
-                ).run()
-            except Exception as e:
-                error_id = capture_exception(e)
-                return Response(
-                    {
-                        "error": f"An issue occured while trying to create external issue. Sentry error ID: {error_id}"
-                    },
-                    status=500,
-                )
```

Comment:
Previously we had an exception catcher in the endpoint and because we determined any errors happening from this block would be our fault we responded here. Now we can use the top level error handler to do that and let the exception go up.